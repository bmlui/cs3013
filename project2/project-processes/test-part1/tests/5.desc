evaluate cd with too many arguments
