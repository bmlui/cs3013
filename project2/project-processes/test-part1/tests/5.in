cd /tmp too-many
exit
