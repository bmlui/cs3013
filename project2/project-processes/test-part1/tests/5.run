PATH="/bad" ./wshell < tests/5.in
