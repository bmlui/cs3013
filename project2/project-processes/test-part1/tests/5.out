test-part1$ cd /tmp too-many
wshell: cd: too many arguments
test-part1$ exit
